api/SWR/CarePackage/useCarePackageApi.js

```diff
@@ -28,7 +28,7 @@ const useCarePackageApi = {
     useGetData(shouldFetch ? `/suppliers${getQueryParamsFromObject({ supplierName })}` : null),
 
   singleSupplier: (supplierId) => useGetData(supplierId ? `/suppliers/${supplierId}` : null, '', {}),
-  singlePackageInfo: (packageId) => useGetData(`${CARE_PACKAGES_URL}/${packageId}`, '', {}),
+  singlePackageInfo: (packageId) => useGetData(packageId ? `${CARE_PACKAGES_URL}/${packageId}` : null, '', {}),
```

Comment:
Maybe we should provide a notification or warning if the packageId is null? what do you think?